isis/src/base/objs/ProcessByBrick/ProcessByBrick.cpp

```diff
@@ -456,8 +456,8 @@ void ProcessByBrick::SetOutputRequirements(int outputRequirements) {
     bool haveInput = PrepProcessCubeInPlace(&cube, &brick);
 
     // Loop and let the app programmer work with the bricks
-    p_progress->SetMaximumSteps(brick->Bricks());
-    p_progress->CheckStatus();
+    //p_progress->SetMaximumSteps(brick->Bricks());
+    //p_progress->CheckStatus();
```

Comment:
Why are these commented out?

Reply:
The progress bar is instantiated on the main application thread.  The processing takes place on the object thread, and when it tries to access the main application thread it will cause seg faults or other weird behavior.  So I commented these out just to get things running, realizing that if we use threads in the future, functionality needs to be added to the ProcessByBrick (or Process class) to facilitate communication between threads.